### PR TITLE
Added 201-documentdb-webapp template

### PR DESCRIPTION
**Pull request by nzthiago**
New template sample for creating a DocumentDB account and a WebApp.

### CODE REVIEW

File: 201-documentdb-webapp/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"databaseAccountName": {
+			"type": "string",
+			"metadata": {
+				"description": "The DocumentDB database account name."
+			}
+		},
+		"databaseAccountTier": {
```
Comment (singhkay) [inline 201-documentdb-webapp/azuredeploy.json]: This should become a variable since it only has one allowed value

**Comment by singhkay** [comment]
@template-deploy-bot test this please

**Comment by nzthiago** [comment]
Good point on the parameter becoming a variable if it only has one allowed value. It was for when/if they add more tiers to DocumentDB but I'd have to update the template when they do anyways

**Comment by singhkay** [comment]
@nzthiago So, I'll wait for you to update the parameter before merging

**Comment by nzthiago** [comment]
@singhkay It's all done

**Comment by singhkay** [comment]
@nzthiago Ah! missed it :) lgtm, merged

**Pull request merged**